pkgs/development/libraries/opencv/4.x.nix

```diff
@@ -44,7 +44,7 @@
 , enableCublas ? enableCuda
 , enableCudnn ? false # NOTE: CUDNN has a large impact on closure size so we disable it by default
 , enableCufft ? enableCuda
-, cudaPackages ? {}
+, cudaPackages ? cudaPackages
```

Comment:
What is the idea behind this construction?

Reply:
will be fixed in #385821